app/actuators/ReactiveList.js

```diff
@@ -102,6 +103,11 @@ export default class ReactiveList extends Component {
 		if (!this.state.showPlaceholder && !this.props.scrollOnTarget) {
 			this.applyScroll();
 		}
+		if (this.state.containerHeight !== this.listContainer.clientHeight) {
+			this.setState({
```

Comment:
Instead of updating the state in componentDidUpdate, try checking if the container height here to see if it is <= 300, if so, apply styles to the container node for PoweredBy (`containerNode.style.display = "none";`)